lib/src/footer/Footer.tsx

```diff
@@ -106,9 +116,23 @@ const BottomContainer = styled.div`
   margin-top: 16px;
 `;
 
-const ChildComponents = styled.div`
+const ChildComponents = styled.div<{ mode: "default" | "reduced" }>`
   min-height: 16px;
```

Comment:
`children` is always responsibility of the user. We do not controlled how the children are shown. The children can be anything, a sentence, a key:value... whatever. So the user is who must control how it is shown using our layout components.